assets/scss/components/elements/blog/_single.scss

```diff
@@ -125,3 +131,172 @@
 	color: #fff !important;
   }
 }
+
+.nv-post-cover {
+  min-height: var(--height);
+  color: var(--color, var(--nv-text-dark-bg));
+  padding: var(--padding);
+  display: flex;
+  position: relative;
+
+  &.has-image{
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center;
+  }
+
+  .nv-title-meta-wrap {
+	display: inline-flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	z-index: 1;
+	&.reversed{
+	  flex-direction: column-reverse;
+	}
+	&.is-boxed{
+	  padding: var(--boxedTitlePadding);
+	  background-color: var(--boxedTitleBackground, var(--nv-light-bg));
```

Comment:
Same thing here `--boxBg` should be enough. Also, let's use background instead of background-color if it doesn't conflict with anything.